pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go

```diff
@@ -200,29 +217,39 @@ func (im *Manager) evalTask(job *Job) error {
 	currentType := latestCond.Type
 
 	if currentType == sednav1.ILJobStageCondWaiting {
-		err := im.loadDeployModel(job)
-		if err != nil {
-			return fmt.Errorf("failed to sync deploy model, and waiting it: %v", err)
-		}
-	}
+		var err error
 
-	if currentType == sednav1.ILJobStageCondWaiting && jobConfig.EvalTriggerStatus == TriggerReadyStatus {
-		payload, err := im.triggerEvalTask(job)
-		if err != nil {
-			klog.Errorf("job(name=%s) complete the %sing phase triggering task failed, error: %v",
-				jobConfig.UniqueIdentifier, jobStage, err)
-			return err
+		err = im.loadDataset(job)
+		if err != nil || jobConfig.Dataset == nil || jobConfig.Dataset.DataSource == nil {
+			return fmt.Errorf("job(name=%s) failed to load dataset, and waiting it: %v",
```

Comment:
a nic: error wrap using `%w` instead of `%v`, see https://blog.golang.org/go1.13-errors#TOC_3.3.

Reply:
fix